app/components/RateJokeForm.tsx

```diff
@@ -0,0 +1,78 @@
+import React from "react";
+import { Button } from "@mando-collabs/tailwind-ui";
+import { LightningBoltIcon } from "@heroicons/react/outline";
+import classNames from "classnames";
+import { ValidatedForm } from "remix-validated-form";
+import { rateJokeFormValidator } from "~/forms/rating-schemas";
+import type { RootLoaderData } from "~/routes/__index";
+
+type MyRating = RootLoaderData["jokeQueueJokes"][number]["myRating"];
+type JokeRatings = RootLoaderData["jokeQueueJokes"][number]["ratings"];
+
+export interface JokeRatingFormProps {
```

Comment:
I would just rename this RateJokeFormProps

Reply:
oh yeah, good catch. I forgot to rename the interface when I renamed the component